Aquarius/Src/Aquarius/Renderer/Renderer.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "Aquarius/Renderer/VertexArray.h"
+#include "Aquarius/Renderer/IndexBuffer.h"
+#include "Aquarius/Renderer/Shader.h"
+
+#include <glm/glm.hpp>
+
+
+namespace Aquarius {
+
+    class Renderer
+    {
+    public:
+        Renderer() = delete;
+        ~Renderer() = delete;
```

Comment:
The intention of this class is static only, so I deleted the default constructor. However, Im not sure if there are still other implicit constructors (copy, move, etc). 

Also, I feel like this is a terrible thing to do?